### ISSUE TITLE
Unable to generate session credentials

### ISSUE THREAD

**Issue by bgs113** [Noovolari/leapp#389]
**Describe the bug**
I'm unable to generate a session using the CLI. I'm using the credential_process method. I started receiving the following error:

```sh
❯ leapp session generate <SESSION_ID>
 ›   Error: command session:generate:<SESSION_ID> not found
```

**Leapp Version**
Leapp Version 0.17.2 (0.17.2)
Leapp Cli Version 0.1.29 (Core: 0.1.131)

**To Reproduce**
1. Start a session either using CLI or GUI
2. Run `leapp session generate <SESSION_ID>` or attempt to use the AWS profile added when starting the session

**Expected behavior**
The session should be generated, instead of returning an error.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: macOS
 - OS Version: Ventura 13.2 (22D49)
 - Leapp Version: 0.17.2

**Additional context**
I tried reinstalling both Leapp and leapp-cli, along with removing `~/.Leapp` and starting over from scratch. I also tried downgrading to 0.17.1, but none of these worked.

**Comment by urz9999** [comment]
Hi @bgs113 did you retrieve the actual session id of the session you want to start? The command you can use is `leapp session list -x` then you can do `leapp session generate` ACTUAL SESSION ID COPIED (e.g. `leapp session generate d07e63ce-a21a-11ed-a8fc-0242ac120002  `). This will generate a set of STS credentials you can use until they expire. However credential process as well as standard credential file methods require you to START a session using `leapp session start`, they only change how Leapp generates and rotates credentials for you.

**Comment by bgs113** [comment]
@urz9999 Yes, the session IDs were just redacted in the examples above. I typically use that for the `credential_process` integration with the AWS CLI, and it had been working until upgrading to leapp-cli 0.1.29. I was able to reproduce the issue on another system with that version as well. For now, I've downgraded to leapp-cli 0.1.28 and pinned that version in homebrew, which is working.

**Comment by kevcube** [comment]
Same issue. cli 0.1.29 (core: 0.1.131)

```
❯ leapp session generate e72d0a62-ce71-4b07-8c11-6f31bccccbc6
 ›   Error: command session:generate:e72d0a62-ce71-4b07-8c11-6f31bccccbc6 not found
```

**Comment by ericvilla** [comment]
We have just released Leapp CLI v0.1.30 which solves this issue with an updated version of the oclif/core package. I'm going to close this issue
